File: app/src/main/java/com/neome/api/ent/base/dto/DeeplinkDataPayloadEntReportShare.java
// neome.ai API - do not change
//

package com.neome.api.ent.base.dto;

import com.neome.api.meta.base.Types.EnumDefnRenderingKind;
import com.neome.api.meta.base.Types.MetaIdLayoutForm;
import com.neome.api.meta.base.dto.DefnForm;
import com.neome.api.meta.base.dto.FormValue;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DeeplinkDataPayloadEntReportShare extends DeeplinkDataPayloadEnt
{
  public DefnForm defnForm;

  public FormValue formValue;

  @Nullable
  public MetaIdLayoutForm outputFormContentLayoutId;

  @Nullable
  public MetaIdLayoutForm outputFormTemplateLayoutId;

  @Nullable
  public Long paperHeight;

  @Nullable
  public EnumDefnRenderingKind paperSize;

  @Nullable
  public Long paperWidth;

  @Nullable
  public String reportLabel;

  public String reportName;
}

File: app/src/main/java/com/neome/api/ent/base/dto/DeeplinkDataPayloadEntReportShare.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.base.dto

import com.neome.api.ent.base.dto.DeeplinkDataPayloadEnt
import com.neome.api.meta.base.dto.DefnForm
import com.neome.api.meta.base.Types.EnumDefnRenderingKind
import com.neome.api.meta.base.dto.FormValue
import com.neome.api.meta.base.Types.MetaIdLayoutForm

interface DeeplinkDataPayloadEntReportShare : DeeplinkDataPayloadEnt
{
  val defnForm: DefnForm
  val formValue: FormValue
  val outputFormContentLayoutId: MetaIdLayoutForm?
  val outputFormTemplateLayoutId: MetaIdLayoutForm?
  val paperHeight: Long?
  val paperSize: EnumDefnRenderingKind?
  val paperWidth: Long?
  val reportLabel: String?
  val reportName: String
}